modules/transport-netty4/src/main/java/org/elasticsearch/transport/netty4/Netty4InternalESLogger.java

```diff
@@ -161,7 +161,7 @@ public boolean isErrorEnabled() {
 
     @Override
     public void error(String msg) {
-        logger.equals(msg);
+        logger.error(msg);
```

Comment:
I pushed 1b660c51275972e01520becafafc82110c8e50a0, can you merge master in to your branch and then I can pull these rest of these in?